core/src/test/java/com/graphhopper/routing/weighting/CustomWeightingTest.java

```diff
@@ -243,6 +244,19 @@ public void testIntEncodedValue() {
         assertEquals(3.70, weighting.calcEdgeWeight(fastEdge, false), 0.01);
     }
 
+    @Test
+    public void unorderedMapShouldFail() {
+        Map map = new HashMap<>();
+        map.put(" > 1.5", 0.2); // allow decimal values in range even for int encoded value
+        CustomModel vehicleModel = new CustomModel();
+        vehicleModel.getPriority().put("lanes", map);
+        try {
```

Comment:
junit 5 has `assertThrows(...)`.

Reply:
Ah, thanks. Old habits are hard to kill :)